ep/build-velox/src/get_velox.sh

```diff
@@ -242,8 +229,16 @@ if [[ "$RUN_SETUP_SCRIPT" == "ON" ]]; then
     echo "Unsupported kernel: $OS"
     exit 1
   fi
+}
+
+prepare_velox_source_code
+
+if [[ "$RUN_SETUP_SCRIPT" == "ON" ]]; then
+  apply_setup_fixes
 fi
 
-apply_compilation_fixes $CURRENT_DIR $VELOX_SOURCE_DIR
+apply_provided_velox_patch "$VELOX_SOURCE_DIR"
```

Comment:
VELOX_HOME is a basic global variable. I think we can just access it inside the function instead of passing the arg.